modules/monitor-osp-disable-helm.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// configuration/monitor-acs.adoc
+:_content-type: PROCEDURE
+[id="monitor-osp-disable-helm_{context}"]
+= Disabling {osp} monitoring for Central services by using Helm
+
+To disable the default monitoring by using Helm, change the configuration options in the `central-services` Helm chart. For more information on configuration options, see the documents in the "Additional resources" section.
```

Comment:
I don't see this section in the preview?

Reply:
Looks like it was removed in 1ba914c971b64df458116a505494806ed747cf88 https://github.com/openshift/openshift-docs/compare/5a3841c5e14c44720d3204a0cad3227ba7f8bca9..1ba914c971b64df458116a505494806ed747cf88